airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/IntegrationConfig.java

```diff
@@ -28,8 +29,13 @@ public static IntegrationConfig spec() {
   }
 
   public static IntegrationConfig check(final Path config) {
+    return check(config, null);
+  }
+
+  public static IntegrationConfig check(final Path config, final Path catalog) {
     Preconditions.checkNotNull(config);
-    return new IntegrationConfig(Command.CHECK, config, null, null);
+    // Not checking catalog for null here because this is in dark launch
```

Comment:
Is this launchDarkly or is dark launch something else?

Reply:
ah dang it haha, if I ever say "dark launch" I just mean "launch with a feature flag to 0% of customers". Old habits die hard.